prow/jobs/jinja/postsubmits/controller_release.jinja2

```diff
@@ -37,4 +37,34 @@
     - ^v[0-9]+\.[0-9]+\.[0-9]+$
 {% endif %}
 
+  - name: {{ service }}-docs-release
+    decorate: true
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    {% for other_service in  aws_services if not other_service == service %}
+    - org: aws-controllers-k8s
+      repo: {{ other_service }}-controller
+      base_ref: main
+    {% endfor %}
+    spec:
+      serviceAccountName: post-submit-service-account
+      containers:
+        - image: public.ecr.aws/aws-controllers-k8s/prow:prow-docs-0.0.1
+          env:
+          - name: GITHUB_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: github-docs-token
+                key: token
+          - name: GITHUB_ACTOR
+            value: "ack-bot"
+          command: ["/build-docs.sh"]
+    run_if_changed: "config/crd/bases/.*"
+    branches:
+    - ^v[0-9]+\.[0-9]+\.[0-9]+$
+    - stable
```

Comment:
Why would we run this for 'stable' branch?
Should It only run for semver releases?

Reply:
Yeah I suppose semver is enough. CRDs should not change from semver to stable, right?

Reply:
That is correct.